api/lib/domain/services/mail-service.js

```diff
@@ -19,6 +19,10 @@ const PIX_NAME_EN = 'PIX - Noreply';
 const HELPDESK_FRENCH_FRANCE = 'https://support.pix.fr';
 const HELPDESK_ENGLISH_SPOKEN = 'https://support.pix.org/en/support/home';
 const HELPDESK_FRENCH_SPOKEN = 'https://support.pix.org';
+const PIX_HOME_NAME_INTERNATIONAL = `pix${settings.domain.tldOrg}`;
+const PIX_HOME_NAME_FRENCH_FRANCE = `pix${settings.domain.tldFr}`;
+const PIX_HOME_URL_ENGLISH_SPOKEN = `${settings.domain.pix + settings.domain.tldOrg}/en-gb`;
+const PIX_HOME_URL_FRENCH_FRANCE = `${settings.domain.pix + settings.domain.tldFr}`;
```

Comment:
Note après écriture : c'est peut-être overkill ce que je suggère là-dessus par rapport au besoin. Le voir + comme une expérience de pensée  ^^

Je m'interroge sur le `/en-gb` est-ce que celui ne devrait pas être en config ? 
Par exemple si il devait avoir à être géré `/fr-be` plus tard il faudrait ajouter des `const` en dur.

Idée dans config.js pour HOME_URL

```js
    domain: {
      tldFr: process.env.TLD_FR || '.fr',
      tldOrg: process.env.TLD_ORG || '.org',
      pix: process.env.DOMAIN_PIX || 'https://pix',
      pixApp: process.env.DOMAIN_PIX_APP || 'https://app.pix',
      pixAppHomeFR: process.env.HOME_PIX_APP_FR || '/',
      pixAppHomeORG: process.env.HOME_PIX_APP_ORG || '/',
      pixAppHomeEN: process.env.HOME_PIX_APP_EN || '/en-gb',
      pixAppHomeBE: process.env.HOME_PIX_APP_BE || '/fr-be',
      pixOrga: process.env.DOMAIN_PIX_ORGA || 'https://orga.pix',
      pixCertif: process.env.DOMAIN_PIX_CERTIF || 'https://certif.pix',
      pix1d: process.env.DOMAIN_PIX1D || 'https://pix1d.pix',
    },
```

et une fonction du style (écrite un peu à l'arrache avant de partir désolé)

```js
function getPixAppHomePage(locale = 'fr') {
  const homepage = settings.domain["pixAppHome"+ locale.toUpperCase()];
  if(homepage) {
    return `${settings.domain.pix + settings.domain.tldOrg + homepage}`
  }

  return `${settings.domain.pix + settings.domain.tldFr + settings.domain.pixAppHomeFR}`
}

///////////////////
// ...

en: {
      ...enTranslations['certification-result-email'].params,
      homeName: PIX_HOME_NAME_INTERNATIONAL,
      homeUrl: getPixAppHomePage('en'),
      title: translate({ phrase: 'certification-result-email.title', locale: 'en' }, { sessionId }),
    },

```

Reply:
je leur ai envoyé un message pour regarder ça :)

Reply:
:+1: sur le principe !